.travis.yml

```diff
@@ -29,11 +29,11 @@ jobs:
     name: "Run tests gRPC example Scala"
 
   - script:
-    - "bin/runSbtJob shopping-cart/shopping-cart-java test"
+    - "bin/runSbtJob shopping-cart/shopping-cart-java docker:publishLocal test"
     name: "Run tests Shopping Cart example Java"
```

Comment:
I guess we're not testing the Maven build in the Java project, but perhaps should. WDYT, @lagom/core?

Reply:
Tracking as https://github.com/lagom/lagom-samples/issues/32.